cyhy/core/geoloc.py

```diff
@@ -39,3 +40,14 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_cidr(self, cidr):
+        has_restricted = False
+        try:
+            response = self.__reader.city(str(cidr[0]))
+            if response.country.name in RESTRICTED_COUNTRIES:
+                print("Warning! %s traced to restricted country: %s" % (cidr, response.country.name))
+                has_restricted = True
+        except AddressNotFoundError:
+            print("CIDR %s not found in geolocation database", cidr)
```

Comment:
I think I would prefer these to print to `sys.stderr` as most of the `print` statements in this library do, but what does everyone else think @cisagov/team-ois ?

Reply:
I think @mcdonnnj's intent was to print _all_ these error messages to `stderr`, not just the one example he pointed out.

Reply:
I did, and I would prefer them to follow the format seen elsewhere in the code base such as https://github.com/cisagov/cyhy-core/blob/79d25173b91c6cd71bd106733f21020c81282e5a/cyhy/db/database.py#L130

Reply:
@mcdonnnj, I'm waiting for this last request to be resolved before re-reviewing.  I'll likely approve if nothing else comes up; @st0rmbl3ss3d has done some useful and quality work here!